modules/overview-of-user-types-and-permissions.adoc

```diff
@@ -53,6 +53,7 @@ The user groups configured in OpenShift, `cluster-admins` and `dedicated-admins`
 * Adding users to the {productname-short} user and administrator groups, if you are using specialized groups.
 * Removing users from the {productname-short} user and administrator groups, if you are using specialized groups.
 * Managing custom environment and storage configuration for users in OpenShift, such as Jupyter notebook resources, ConfigMaps, and persistent volume claims (PVCs).
+* Managing data science pipeline applications and pipeline servers from the {productname-short} dashboard, or from the command-line interface (CLI), for any data science project. 
```

Comment:
minor: should pipeline server come first?
"Managing data science pipeline servers and pipeline applications from..."